src/main/java/de/hysky/skyblocker/config/configs/SlayersConfig.java

```diff
@@ -3,12 +3,26 @@
 import dev.isxander.yacl3.config.v2.api.SerialEntry;
 
 public class SlayersConfig {
+    @SerialEntry
+    public HighlightSlayerEntities highlightMinis = HighlightSlayerEntities.OFF;
+
+    @SerialEntry
+    public HighlightSlayerEntities highlightBosses = HighlightSlayerEntities.OFF;
+
+    public enum HighlightSlayerEntities {
+        OFF, GLOW, HITBOX
+    }
+
     @SerialEntry
     public EndermanSlayer endermanSlayer = new EndermanSlayer();
 
     @SerialEntry
     public VampireSlayer vampireSlayer = new VampireSlayer();
 
+    @SerialEntry
+    public BlazeSlayer blazeSlayer = new BlazeSlayer();
```

Comment:
I would have blaze slayer config before vampire to be consistent with the in game order.